### PR TITLE
Create pick

### PR DESCRIPTION
**Pull request by mikeymurph77**
Adding the pick model, associations, and tests.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -36,8 +46,10 @@
   add_index "seasons", ["start_date"], name: "index_seasons_on_start_date", unique: true, using: :btree
 
   create_table "teams", force: :cascade do |t|
-    t.string "name",     null: false
-    t.string "location", null: false
+    t.string   "name",       null: false
+    t.string   "location",   null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
```
Comment (mikeymurph77) [inline db/schema.rb:52]: Will remove timestamps on a new branch once this branch is merged.

File: app/models/pick.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class Pick < ActiveRecord::Base
+  validates :user_id, presence: true
```
Comment (edwardloveall) [inline app/models/pick.rb]: I just tested this on my computer and you can remove the `_id` suffix here and in the test and all tests run fine. Slightly cleaner IMO too.

File: db/migrate/20160215223349_create_picks.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+class CreatePicks < ActiveRecord::Migration
+  def change
+    create_table :picks do |t|
+      t.belongs_to :user, null: false
```
Comment (edwardloveall) [inline db/migrate/20160215223349_create_picks.rb:4]: A shortcut here is:

``` ruby
t.belongs_to :user, null: false, index: true
```

What you did is fine though. Maybe even better as it's very clear what is being added to the database. Shortcut for your knowledge.

File: db/schema.rb (hunk 1 above)
Comment (edwardloveall) [inline db/schema.rb:52]: I think you should keep them. Timestamps (in particular) are incredibly valuable for tracking down bugs, creating analytics, sorting results, etc, etc. There's almost never a good reason to remove them. If you think there it we should talk about it. `The app doesn't directly use timestamps on the Team model` isn't a good enough reason.

File: app/models/pick.rb (hunk 2 above)
Comment (mikeymurph77) [inline app/models/pick.rb]: Great! I removed the `_id` suffix from other attributes throughout the models and tests in this branch as well.

File: db/migrate/20160215223349_create_picks.rb (hunk 3 above)
Comment (mikeymurph77) [inline db/migrate/20160215223349_create_picks.rb:4]: I like your method. I think it's clear enough. I'll start using this in my migrations in the future

File: db/schema.rb (hunk 1 above)
Comment (mikeymurph77) [inline db/schema.rb:52]: I agree.

**Comment by edwardloveall** [comment]
Looks good to me. :shipit:

**Pull request merged**